client/src/views/Register.js

```diff
@@ -9,10 +9,33 @@ const Register = (props) => {
     const [confirmReg, setConfirmReg] = useState("");
     const [user, setUser] = useState({
         sjsuid: "",
+        securityQuestion: "Which is your favorite class?", // default selected val
+        securityAnswer: "",
         password: "",
         confirmPassword: "",
     });
 
+    const questions = [
+        { label: "Which is your favorite class?", value: "Which is your favorite class?" },
+        { label: "In which city were you born?", value: "In which city were you born?" },
+        { label: "What's the name of your first pet?", value: "What's the name of your first pet?" },
+        { label: "What's your childhood best friend's name?", value: "What's your childhood best friend's name?"},
+    ];
```

Comment:
Is there a way to retrieve labels + values directly from the enum from User.model?